pyEpiabm/pyEpiabm/sweep/initial_place_sweep.py

```diff
@@ -67,7 +67,12 @@ def __call__(self, *args):
                                               power_law_params=power_list)
 
                 elif place.place_type.name == "CareHome":
-                    helper.update_place_group(place)
+                    person_list_use = person_list.copy()
+                    helper.update_carehome_group(place,
+                                                 group_size=ave_group_size,
+                                                 person_list=person_list_use,
```

Comment:
```suggestion
                    helper.update_carehome_group(place,
                                                 group_size=ave_group_size,
                                                 person_list=person_list.copy(),
```
If we need to pass a copy, its probably easier to create it on the fly